### PR TITLE
Issue #12229: Renamed Checker-Frameework to Checker-checker

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix: #12229

### CODE REVIEW

File: .github/workflows/checker-framework.yml [hunk 1]
```diff
@@ -12,14 +12,14 @@ jobs:
       fail-fast: false
       matrix:
         profile:
-          - checker-framework-nullness-optional-interning
```
Comment (romani) [inline .github/workflows/checker-framework.yml:15]: good step, but we need to change line #1 also
`name: Checker-Framework` to `name: Checker`

**Review by romani — CHANGES_REQUESTED** [review]
item:

File: .github/workflows/checker-framework.yml (hunk 1 above)
Comment (romani) [reply .github/workflows/checker-framework.yml:15]: put name to job to remove yml file print 
```
jobs:
  checker-framework:
    name: analyse
```

File: .github/workflows/checker-framework.yml [hunk 2]
```diff
@@ -12,14 +12,14 @@ jobs:
...
-          - checker-framework-methods-resource-fenum
-          - checker-framework-lock-tainting
-          - checker-framework-index
-          - checker-framework-formatter
-          - checker-framework-signature-gui-units-init
-          - checker-framework-regex-property-key-compiler-message
-          - checker-framework-purity-value-returns
+          - checker-nullness-optional-interning
+          - checker-methods-resource-fenum
+          - checker-lock-tainting
```
Comment (Vyom-Yadav) [inline .github/workflows/checker-framework.yml:17]: You need to change the `id` of profiles too, you can find the profiles here:
https://github.com/checkstyle/checkstyle/blob/a3224dafe53f501d3ea91f4eb3bd6e30e3c8e4b2/pom.xml#L2404-L2407

File: .github/workflows/checker-framework.yml (hunk 1 above)
Comment (NitishKumar06) [reply .github/workflows/checker-framework.yml:15]: I got it! May I know how changing the word 'analyse' to 'job' would solve this failure in CI status?

File: .github/workflows/checker-framework.yml (hunk 2 above)
Comment (NitishKumar06) [reply .github/workflows/checker-framework.yml:17]: I see. So it was included for all those files that includes "checker-framewor". Right?

File: .github/workflows/checker-framework.yml (hunk 1 above)
Comment (romani) [reply .github/workflows/checker-framework.yml:15]: May be somewhere in doc of GitHub.
But you can check all names that we have in CI list and analyse corresponding yml file and you will see this.

**Comment by romani** [comment]
Should be single commit in PR

**Comment by NitishKumar06** [comment]
@romani It's done now!

**Comment by Vyom-Yadav** [comment]
@NitishKumar06 Please make CI green.

File: .github/workflows/checker-framework.yml (hunk 2 above)
Comment (romani) [reply .github/workflows/checker-framework.yml:17]: Yes, all such names are from pom.xml

**Comment by romani** [comment]
I do not see Checker CI in list - please recheck why they failed to start.

**Comment by NitishKumar06** [comment]
@romani it's compilation error. I'm still trying to figure out how did this error generated since I've raised a PR after this PR and the build for those were completely successful.

errors

**Comment by NitishKumar06** [comment]
@romani @rnveach After making the changes, I'm getting the following error: `[ERROR] [checkstyle] [ERROR] C:\Users\justn\Desktop\Open Source\checkstyle\.ci\checker-framework-suppressions\checker-framework-methods-resource-fenum-suppressions.xml:97: Line should not be longer than 100 symbols [lineLength]` for most no. of lines:(

**Comment by romani** [comment]
It might be good for you to fix only job name that is rendered in web browser, all other can stay same.

**Comment by NitishKumar06** [comment]
@romani By looking at the file changes, is adding job mandatory? Kindly give a review on which no. of line job is to be added.

**Comment by romani** [comment]
I think you can complete this issue by
https://github.com/checkstyle/checkstyle/pull/12269#discussion_r990642927

or as you did:
```
jobs:
  checker:
    strategy:
```

**Comment by NitishKumar06** [comment]
@romani Look at the file changes, I've applied the same changes as mentioned in the comment. Isn't it?

**Comment by romani** [comment]
I saw them, if you want to change more that one file, you have update all places,  please follow CI failures it will guide what to fix.

**Comment by NitishKumar06** [comment]
So we don't actually need to care for jobs?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by romani** [comment]
we need to shorten name of job in CI status to make ease to read on phone, nothing else.

**Comment by NitishKumar06** [comment]
@romani I know this may sound childish, but Could you please explain what you actually mean by shortening the name of job?

As you can see in the files, I just put the following contents you mentioned :

`jobs:
  checker-framework:
    name: analyse`

**Comment by romani** [comment]
Make it:
```
jobs:
  checker:
    name: analyse
```

**Comment by romani** [comment]
The way how it looks at phone
Screenshot_20221014-143715

What Checker is failed?

File: .ci/checker-framework.groovy [hunk 3]
```diff
@@ -7,8 +7,8 @@ import groovy.xml.XmlUtil
 import java.util.regex.Matcher
 import java.util.regex.Pattern
 
-@Field static final String USAGE_STRING = "Usage groovy " +
-        ".${File.separator}.ci${File.separator}checker-framework.groovy" +
+@Field final String USAGE_STRING = "Usage groovy " +
```
Comment (romani) [inline .ci/checker-framework.groovy:10]: Keep static, it not related to your issue

File: .github/workflows/checker-framework.yml [hunk 4]
```diff
@@ -32,6 +32,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Execute checker framework
-        run: groovy ./.ci/checker-framework.groovy ${{ matrix.profile }}
+        run: groovy ./.ci/checker.groovy ${{ matrix.profile }}
```
Comment (romani) [inline .github/workflows/checker-framework.yml:35]: Please keep file name same as before.

File: .github/workflows/checker-framework.yml [hunk 5]
```diff
@@ -32,6 +32,7 @@ jobs:
...
         env:
+      - name: analyse
           MAVEN_OPTS: "-Xmx4g"
```
Comment (romani) [inline .github/workflows/checker-framework.yml:38]: This is not a place where we need it. This name is shown in log page, there are plenty of space

File: config/checkstyle_non_main_files_suppressions.xml [hunk 6]
```diff
@@ -98,7 +98,7 @@
   <suppress id="lineLength" files=".ci[\\/]error-prone-suppressions[\\/].*"/>
 
   <!-- this file cannot be line wrapped, It contains a list of suppressions -->
-  <suppress id="lineLength" files=".ci[\\/]checker-framework-suppressions[\\/].*"/>
+  <suppress id="lineLength" files=".ci[\\/]checker-suppressions[\\/].*"/>
```
Comment (romani) [inline config/checkstyle_non_main_files_suppressions.xml:101]: Let's keep orginal name

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: .ci/checker-framework.groovy (hunk 3 above)
Comment (NitishKumar06) [reply .ci/checker-framework.groovy:10]: Okay!

File: .github/workflows/checker-framework.yml (hunk 5 above)
Comment (NitishKumar06) [reply .github/workflows/checker-framework.yml:38]: So on which file the following code should be put in?

File: config/checkstyle_non_main_files_suppressions.xml (hunk 6 above)
Comment (NitishKumar06) [reply config/checkstyle_non_main_files_suppressions.xml:101]: Alright!

**Comment by romani** [comment]
took to much time to discuss, I made PR https://github.com/checkstyle/checkstyle/pull/12324#issuecomment-1287811449

**Pull request closed without merging**